src/newfight.cpp

```diff
@@ -1188,8 +1232,17 @@ void hit_with_multiweapon_toggle(struct char_data *attacker, struct char_data *v
     act( rbuf, 1, att->ch, NULL, NULL, TO_ROLLS );
     
     // If the attacker dies from recoil, bail out.
-    if (damage(att->ch, att->ch, convert_damage(staged_dam), TYPE_HIT, FALSE))
+    if (damage(att->ch, att->ch, convert_damage(staged_dam), TYPE_HIT, FALSE)) {
```

Comment:
Same issue re: attacker dying and extracting the defending quest target.

Reply:
Roger